scroll-animations-1/Overview.bs

```diff
@@ -605,15 +606,20 @@ spec: cssom-view-1; type: dfn;
 			1. Set the {{ScrollTimeline/source}} of |timeline|
 				to the {{ViewTimeline/subject}}’s
 				nearest ancestor [=scroll container=] element.
+
+			1.  If a {{DOMString}} value is provided as an |inset|,
+				parse it as a <<'view-timeline-inset'>> value;
+				if a sequence is provided,
+				the first value represents the start inset
+				and the second value represents the end inset.
```

Comment:
What if it's a sequence of length one? Should we follow the same convention as we use for parsing that a single value expands to both? What about a sequence of length 0? Should we throw a TypeError?

Reply:
Agree that repeating would be nice, and throwing a TypeError for anything else.